File: Programmers/pro_42576_완주하지못한선수.py
from collections import defaultdict
def solution(participant, completion):
    dict_cnt = defaultdict(int)
    for i in participant:
        dict_cnt[i]+=1
    for i in completion:
        dict_cnt[i]-=1
    for i in participant:
        if(dict_cnt[i]==1):
            return i
    answer = ''
    return answer

File: Programmers/pro_42576_완주하지못한선수.js
function solution(participant, completion) {
    const participantDict = {}
    participant.forEach(name => participantDict[name] ? participantDict[name] += 1 : participantDict[name] = 1)
    completion.forEach(name => participantDict[name] -= 1)
    return Object.entries(participantDict).find(([_, value]) => value > 0)[0]
}